zip-0302.rst

```diff
@@ -0,0 +1,81 @@
+::
+
+  ZIP: 302
+  Title: Standardized Memo Field Format
+  Owner: Jack Grigg <jack@electriccoin.co>
+  Original-Author: Jay Graber
+  Status: Draft
+  Category: Standards
+  Created: 2017-02-08
+  License: MIT
+
+
+Abstract
+========
+
+This ZIP describes a proposed specification for a standardized format for displaying content within the encrypted memo field of shielded transactions.
+
+Motivation
+==========
+
+A well-defined standard for formatting content within the encrypted memo field would help expand the use cases by providing a structure for different types of data. Users and third-party services could benefit from a standardized formatting convention that defines the type and length of the data contained within.
+
+A proposed common use case could be a standard encoding for sender’s preferred return address.
+
+Specification
+===============
+
+Section 5.5 of the Zcash protocol specification [#protocol-notept]_ defines three cases
+for the encoding of a memo field:
+
+* a UTF-8 human-readable string [#Unicode]_, padded by appending zero bytes; or
+* the byte ``0xF6`` followed by 511 ``0x00`` bytes, indicating "no memo"; or
+* any other sequence of 512 bytes starting with a byte value ``0xF5`` or greater
+  (which is therefore not a valid UTF-8 string), as specified in ZIP 302.
+
+This ZIP refines the specification of the third case.
+
++ If the first byte (byte 0)'s value is 0xF4 or smaller, then the reader MUST:
```

Comment:
Nit: the apostrophe on `first byte (byte 0)'s` read a little awkwardly. Suggested tweak:

```suggestion
+ If the first byte (byte 0) has a value of 0xF4 or smaller, then the reader MUST:
```